### DISCUSSION TITLE
Touch screen mode says 'no printer connected' but other mode works fine.

### DISCUSSION THREAD

**Discussion by KVet92** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2605] [Q&A]
Upgraded my Ender 3 to a BIGTREETECH SKR Mini E3 V3.0 Control Board + TFT35 E3 V3.0.1 Touch Screen and everything is connected properly, but when I boot it up, it throws this error at the top when in the touch screen mode, but the other mode works perfectly fine. Firmware on both is all up to date, does anyone have any suggestions?

![photo_2022-09-19_15-35-53](https://user-images.githubusercontent.com/6521069/191101308-ff724e86-b671-4591-ac8a-fc1dd2bcd45c.jpg)

**Comment by oldman4U** [comment]
Read the Read Me for all possible answers.

**Comment by radek8** [comment]
Upload photos of wiring and configuration files.
